src/components/MarkdownProvider/components/Usage.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { MD } from '@zendeskgarden/react-typography';
+
+const StyledUsage = styled.div`
+  @media (min-width: ${p => p.theme.breakpoints.sm}) {
+    display: flex;
+  }
+`;
+
+export const Usage: React.FC = props => <StyledUsage {...props} />;
+
+export interface IUsageSectionProps {
+  type: string;
+}
+
+export const UsageSection: React.FC<{
+  type: string;
```

Comment:
Can you come up with a `boolean` prop name? Ideally for the negative rendering. i.e. `isNotRecommended`. Defer to @rossmoody for naming help :wink:

Reply:
Actually, the preferred treatment would be to follow @austin94's [suggestion](https://github.com/zendeskgarden/website/pull/34#discussion_r423270284) on `BestPractice` and add first-class `<Use>` and `<Misuse>` components.